src/web/app/components/question-text-with-info/question-text-with-info.component.spec.ts

```diff
@@ -30,4 +32,34 @@ describe('QuestionTextWithInfoComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should not show control link and question details for TEXT questions', () => {
+    component.questionDetails.questionType = FeedbackQuestionType.TEXT;
+    const questionDetailControlLink: any = fixture.debugElement.query(By.css('a'));
+    const infoWrapper: any = fixture.debugElement.query(By.css('div'));
+
+    expect(questionDetailControlLink).toBeNull();
+    expect(infoWrapper).toBeNull();
+  });
+
+  it('should show control link when question type is not TEXT', () => {
+    component.questionDetails.questionType = FeedbackQuestionType.MCQ;
```

Comment:
Same here. (and this will be the problem I have mentioned --- the internal structure is of question type text while the type is MCQ)